frontend/src/utils/helpers/materials/extractMaterial.ts

```diff
@@ -0,0 +1,17 @@
+import { ContentElementType, IContentElement } from 'types/course';
+import { optimizeLink } from '../videoPlayer/videoLink';
```

Comment:
```suggestion
import { optimizeLink } from 'utils/helpers/videoPlayer/videoLink';
```

Reply:
Here we should only  separate absolute and relative imports, yeah. But replacing relative with absolute is not really good idea IMHO, cause both files located at `utils` folder. We will just make import longer.